internal/attestation/measurements/measurement-generator/generate.go

```diff
@@ -32,6 +32,9 @@ import (
 	"golang.org/x/tools/go/ast/astutil"
 )
 
+// this tool is used to generate hardcoded measurements for the enterprise build.
+// Measurements are embedded in the constellation cli.
```

Comment:
```suggestion
// This tool is used to generate hard-coded measurements for the enterprise build.
// Measurements are embedded in the Constellation CLI.
```
This should be a package comment.